webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -600,14 +662,15 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
     /**
      * Gets all IRI's of all default object access permissions defined inside a project.
      *
-     * @param projectIri     the IRI of the project.
-     * @param requestingUser the [[UserADM]] of the requesting user.
+     * @param projectIri        the IRI of the project.
+     * @param requestingUser    the [[UserADM]] of the requesting user.
+     * @param apiRequestID      the API request ID.
      * @return a list of IRIs of [[DefaultObjectAccessPermissionADM]] objects.
      */
-    private def defaultObjectAccessPermissionsForProjectGetRequestADM(projectIri: IRI, requestingUser: UserADM): Future[DefaultObjectAccessPermissionsForProjectGetResponseADM] = {
-
-        // FIXME: Check user's permission for operation (issue #370)
-
+    private def defaultObjectAccessPermissionsForProjectGetRequestADM(projectIri: IRI,
+                                                                      requestingUser: UserADM,
+                                                                      apiRequestID: UUID
+                                                                     ): Future[DefaultObjectAccessPermissionsForProjectGetResponseADM] = {
```

Comment:
Still needs a check if the user is project admin in the project that the permission is about.

Reply:
this check is added to the validation of messages.